definitions/firefox_desktop.toml

```
@@ -1255,6 +1253,43 @@ data_source = "main"
 friendly_name = "SSL Loads Probe Ratio"
 description = "Ratio of clients that have the http_pageload_is_ssl_ratio_v1 probe"
 
+[metrics.sponsored_tiles_disabled]
+select_expression = '{{agg_any("sponsored_tiles_disable_count > 0")}}'
+data_source = "sponsored_tiles_clients_daily"
+friendly_name = "Sponsored Tiles Disabled Event"
+description = "Boolean that identifies clients that disabled sponsored tiles during experiment"
```

Sponsored Tiles Daily:
`+` includes Mobile and Desktop
`-` based off of activity stream, which will be deprecated with pingcentre
`?` you want to use these [business filters](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql#L142-L165) to only return data for these clients

Newtab Interactions:
`-` Desktop only, no Mobile
`+` Consistency with other Newtab metrics: [ref 1](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml#L1123), [ref 2](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml#L1004)
`?` No business filters

```suggestion
[metrics.newtab_tiles_enabled]
data_source = "newtab_interactions"
select_expression = "COALESCE(MAX(CAST(topsites_sponsored_enabled AS INT)), 0)"
friendly_name = "Sponsored Tiles Enabled"
type = "scalar"
description = "Whether or not Sponsored tiles are enabled on the New Tab."
```

---

I think the business filters are fine for now, given that it covers all of the markets we're interested in. I used the `sponsored_tiles_clients_daily` source because it records the dismissal event itself, and we can measure if there's a difference in that event between branches. Although, newtab, too, would record the same, assuming the proportion of clients in each branch with that pref set is equal. Do you have thoughts on which should be used, the event or the pref?

---

Actually, I'm not seeing the `topsites_sponsored_enabled` field in the newtab interactions table, even though the data is collected in the [ping](https://dictionary.telemetry.mozilla.org/apps/firefox_desktop/metrics/topsites_sponsored_enabled). I guess the probe was implemented, but not yet added to the newtab interactions table. For your Thursday deadline, let's go with Sponsored Tiles Clients Daily then.

My thought was: If the % of clients that don't have the pref set is low, then go with the pref.